### ISSUE TITLE
selector of the directive

### ISSUE THREAD

**Issue by darklinki** [palantir/tslint#3109]
### Bug Report

- __TSLint version__: 5.2.0
- __TypeScript version__:
- __Running TSLint via__: Phpstorm

#### TypeScript code being linted
```ts
// @Directive({ selector: '[shiftHeight]' })
```

with `tslint.json` configuration:

```json

```

#### Actual behavior
TS2515:The selector of the directive "JobHeightDirective" should have one of the prefixes "nga, ngd, ngt" (https://goo.gl/uacjKR) (directive-selector
#### Expected behavior
https://angular.io/guide/styleguide#!#02-08 Based on the style guide it seems to be right how I wrote it cause its in the component shift. Cant find anything about nga, ngd, ngt in the style guide

**Comment by ajafff** [comment]
That's not a tslint core rule. This issue should be opened over there https://github.com/mgechev/codelyzer
